changelog/63601.added.md

```diff
@@ -0,0 +1 @@
+systemd daemon is reloaded, added in service state and module docs.
```

Comment:
FYI, we don't need changelogs for doc changes. 😄

Reply:
@barbaricyawps Noted